src/Api/Serializer/AbstractSerializer.php

```diff
@@ -43,7 +43,7 @@ abstract class AbstractSerializer extends BaseAbstractSerializer
     /**
      * @var callable[]
      */
-    protected static $mutators = [];
+    protected static $customAttributes = [];
```

Comment:
`customAttributes` doesn't feel plural enough IMO. Maybe something like `attributeMutator`?

Reply:
wouldn't it be `attributeMutators` then? since it's many

Reply:
Yeah that sounds good!